### ISSUE TITLE
Feature request: Search history

### ISSUE THREAD

**Issue by ZeRoZeRoRaZ** [yuliskov/SmartTubeNext#34]
Please implement search history.

**Comment by rodalpho** [comment]
Agree, I would find this very useful. I use the search history as bookmarks for types of videos I want to watch in the future.

**Comment by CiscoFever** [comment]
Absolutely agree, I'm still using the old unsupported SmartYouTubeTV as it does provide a search history of your last 10 searches as 'guest' (no login reqiured).

**Comment by yenic** [comment]
While I've moved to Next, I know people that insist on still using the old app just for this feature (search terms saved while logged out). @yuliskov I hope it's added because I'm the one that helps keep the old app limping along. 🙏

Also mentioned in #581.
